### PR TITLE
solution

### PR DESCRIPTION
**Pull request by lefteris-L**
[DEMO LINK](https://lefteris-l.github.io/react_goods-selector/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +19,98 @@ export const goods = [
...
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              return (
+                <tr
+                  data-cy="Good"
```
Comment (nurdanokcu) [inline src/App.tsx:79]: I couldn't find a key added to the `<tr>...</tr>`

**Review by nurdanokcu — APPROVED** [review]
Amazing work :100:

File: src/App.tsx (hunk 1 above)
Comment (lefteris-L) [reply src/App.tsx:79]: My bad

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +19,99 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = { selectedGood: string };
```
Comment (naziamoff) [inline src/App.tsx:22]:
```suggestion
type State = { 
  selectedGood: string 
};
```

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +19,99 @@ export const goods = [
...
+
+export class App extends Component<{}, State> {
+  state = { selectedGood: 'Jam' };
+
+  isSelected = (item: string) => (
```
Comment (naziamoff) [inline src/App.tsx:27]: from its name it should be clear that it is a function
```suggestion
  checkIsSelected = (item: string) => (
```

**Review by naziamoff — APPROVED** [review]
great job! A couple of suggestions

**Pull request closed without merging**